content/press.adoc

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: "Jenkins Press Information"
+---
+
+
+If you're a journalist, blogger or anybody else who might consider themselves
+"press" this page should help you refer to, explain or otherwise cite the
+Jenkins project correctly. If this page does not sufficiently answer a question
+you may have, or you're seeking comment, please reach out to one of our <<Press Contacts>> below.
+
+
+== About Jenkins
+
+
+.Elevator Pitch
+""
+Jenkins is an open source automation server which enables developers around the
+world reliably build, test and deploy their software
```

Comment:
Hm, it reads fine without 'to' as well. :gb:

Period at the end probably isn't so important, as this is just one sentence being shown on its own in a box (but yeah, that's arguably inconsistent with the "Blurb" box).